src/script/lua_api/l_env.cpp

```diff
@@ -78,7 +80,9 @@ void LuaABM::trigger(ServerEnvironment *env, v3s16 p, MapNode n,
 	lua_pushnumber(L, active_object_count);
 	lua_pushnumber(L, active_object_count_wider);
 
-	PCALL_RESL(L, lua_pcall(L, 4, 0, errorhandler));
+	int result = lua_pcall(L, 4, 0, errorhandler);
+	if (result)
+		scriptIface->scriptError(result, "LuaABM::trigger");
```

Comment:
You don't get a warning if the function gets renamed. We still had lots of copy paste in the networking code, and the functions got renamed without the logging getting fixed.